diff --git a/plugin_test/utils/jenkins/system_tests.sh b/plugin_test/utils/jenkins/system_tests.sh
index 1e81aa5..45c56a7 100755
--- a/plugin_test/utils/jenkins/system_tests.sh
+++ b/plugin_test/utils/jenkins/system_tests.sh
@@ -563,7 +563,7 @@ RunTest() {
         OPTS="${OPTS} ${TEST_OPTIONS}"
     fi
 
-    clean_old_bridges
+    #clean_old_bridges
 
     # run python test set to create environments, deploy and test product
     if [ "${DRY_RUN}" = "yes" ]; then
@@ -595,9 +595,9 @@ RunTest() {
 
 
     # Configre vcenter nodes and interfaces
-    setup_net $ENV_NAME
-    clean_iptables
-    revert_ws "$WORKSTATION_NODES" || { echo "killing $SYSTEST_PID and its childs" && pkill --parent $SYSTEST_PID && kill $SYSTEST_PID && exit 1; }
+    #setup_net $ENV_NAME
+    #clean_iptables
+    #revert_ws "$WORKSTATION_NODES" || { echo "killing $SYSTEST_PID and its childs" && pkill --parent $SYSTEST_PID && kill $SYSTEST_PID && exit 1; }
 
     echo waiting for system tests to finish
     wait $SYSTEST_PID
